### ISSUE TITLE
Wrong error message for delete invalid index

### ISSUE THREAD

**Issue by cleowenxuan** [cleowenxuan/pe#5]
"delete 0" should mention wrong index and not wrong command format for the error message so that users are able to understand what is wrong with their input, instead of trying to find out if their format for the command is wrong.

![Screenshot 2024-04-19 at 4.52.21 PM.png](https://raw.githubusercontent.com/cleowenxuan/pe/main/files/0e91f3ae-f417-4c73-8591-b8a5bd9b3e69.png)

**Comment by nus-pe-script** [comment]
# Team's Response
In the screenshot provided, it is already stated that `PATIENT_INDEX` must be a positive integer. You might be right that the error message is too wordy, but the correct error is still identified and highlighted to the user.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [x] I disagree

**Reason for disagreement:** Referenced from the CS2103T Textbook, "error messages can be correct but not specific enough" can be counted as feature flaws. In this case, 0 is technically a number and the command is in the correct format. As such, it should give the error mentioning 'wrong index' and not 'wrong command format'. 
![Screenshot 2024-04-25 at 2.32.13 PM.png](https://raw.githubusercontent.com/cleowenxuan/pe/main/files/24b270b7-b31e-4a11-84ae-82e962ee9521.png)

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [x] I disagree

**Reason for disagreement:** This feature flaw is of `Low` severity and not `VeryLow` because it displays error that is not specific enough and is not a cosmetic issue.

<catcher-end-of-segment><hr>
